referrer-policy/README.md

```diff
@@ -58,6 +59,17 @@ Run tests under path: ```/referrer-policy```.
 
 Click start.
 
+## Options for generating tests
+
+The generator script ```./generic/tools/generate.py``` has two targets: ```debug``` and ```release```.
+When generating for ```debug```, the produced tests will contain more verbosity and sanity checks.
+Using the release for the target will produce tests using a template for optimizing size and performance.
+The release template is intended for the official web-platform-tests and possibly other test suites.
+No sanity checking is done in release mode.
+
+
+Note that **debug** is the default target when invoking ```generate.py```.
```

Comment:
As above, this seems backwards, given that we don't actually want folks to check in the debug build.